frontend/src/component/commandBar/CommandBar.tsx

```diff
@@ -194,13 +194,18 @@ export const CommandBar = () => {
         },
     );
     useKeyboardShortcut({ key: 'Escape' }, () => {
-        if (searchContainerRef.current?.contains(document.activeElement)) {
-            searchInputRef.current?.blur();
-        }
+        setShowSuggestions(false);
     });
     const placeholder = `Command bar (${hotkey})`;
 
     useOnClickOutside([searchContainerRef], hideSuggestions);
+    const onKeyDown = (event: any) => {
```

Comment:
Couldn't find a good event type for this that let me use event.key and event.keyCode, KeyboardEvent would but isn't compatible with the onKeyDown prop